inc/config.class.php

```diff
@@ -56,14 +56,14 @@ class PluginFlyvemdmConfig extends CommonDBTM {
    const WIZARD_WELCOME_END = 1;
 
    // first and last steps of requirement pages of wizard
-   const WIZARD_REQUIREMENT_BEGIN = 100;
-   const WIZARD_REQUIREMENT_END = 107;
+   const WIZARD_REQUIREMENT_BEGIN = 2;
+   const WIZARD_REQUIREMENT_END = 9;
 
    // first and last steps of the MQTT pages of wizard
-   const WIZARD_MQTT_BEGIN = 108;
-   const WIZARD_MQTT_END = 111;
+   const WIZARD_MQTT_BEGIN = 10;
+   const WIZARD_MQTT_END = 13;
 
-   const WIZARD_FINISH = -1;
+   const WIZARD_FINISH = 14;
```

Comment:
Numbers were chosen to make easier to add pages in the wizard without changing all constants. If we add a page in the MQTT part for example, we don't need to change all parts occuring after it.

WIZARD_FINISH = -1 to also not bother about this value when we change the steps count.

Reply:
I agree with you, but going from step 1 to step 100 is confusing for users

Reply:
how do you imagine this part